source/tools/testConsistency/android-renderer/AndroidDriver/app/src/test/java/com/example/androiddriver/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.example.androiddriver;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
```

Comment:
What's the point of this file? Do we need it? Where are the actual tests again?

Reply:
They are under the directory android test. This is an unnecessary test file that is created by android studio by default. i can get rid of it